diagnostic.md

```diff
@@ -2,76 +2,86 @@
 
 Place your responses inside the fenced code-blocks where indicated by comments.
 
-
 What is the purpose of a backend?
 
-```bash
-// your response here
+```md
+To store and manipulate data for your application.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
-```bash
-// your response here
+```md
+model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
-```bash
-// your response here
+```md
+controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
-```bash
-// your response here
+```md
+Handling views on the front-end allows us to create single page apps as well
+as access the same database through multiple apps (web app, desktop app, mobile
+app, etc) through the same API.
 ```
 
 What does C.R.U.D stand for?
 
-```bash
-// your response here
+```md
+Create Read Update Destroy
 ```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
-```bash
-// your response here
+```md
+controller
 ```
+
 List at least 5 standard actions that C.R.U.D corresponds to?
 
-```bash
-// your response here
+```md
+GET INDEX POST PATCH DELETE
```

Comment:
These are request types, I was looking for controller actions. create, show, index, update, destroy